runtime/near-vm-runner/src/prepare.rs

```diff
@@ -109,301 +34,13 @@ fn validate_contract(code: &[u8], config: &VMConfig) -> Result<(), PrepareError>
 ///
 /// The preprocessing includes injecting code for gas metering and metering the height of stack.
 pub fn prepare_contract(original_code: &[u8], config: &VMConfig) -> Result<Vec<u8>, PrepareError> {
-    validate_contract(original_code, config)?;
+    prepare_v1::validate_contract(original_code, config)?;
```

Comment:
Should move this into `prepare_contract` for each version (even if `v0` continues calling `v1`'s `validate_contract`)